integration-tests/fixtures/generate-bundle-stats/vite/vite-v4-config-template.ts

```diff
@@ -0,0 +1,33 @@
+export const configV4 = ({
+  id,
+  status,
+}: {
+  id: string;
+  status: number;
+}) => `import { codecovVitePlugin } from "@codecov/vite-plugin";
+import path from "path";
```

Comment:
This feels hard to add more test cases for, and gets harder to review because the snippet doesn't get syntax highlighted.

Could we make these templates actual js files? Then you can either 

a) do `fs.readFile` to read the config template and do a string replace to generate the correct `apiUrl`.
b) set the api url to be a environmental variable which you insert in when running shall command. 

I like b) because environmental variables feels easier to understand for others who are looking at codebase.

Reply:
Updated